automation/src/test/scala/SignInSpec.scala

```diff
@@ -0,0 +1,31 @@
+import org.broadinstitute.dsde.firecloud.auth.{AuthToken, AuthTokens}
+import org.broadinstitute.dsde.firecloud.pages.WebBrowserSpec
+import org.broadinstitute.dsde.firecloud.{CleanUp, Config}
+import org.scalatest._
+
+class SignInSpec extends FreeSpec with WebBrowserSpec with CleanUp {
+
+  implicit val authToken: AuthToken = AuthTokens.harry
+
+  "A user" - {
+    "with a registered account" - {
+
+      "should be able to log out and log in as another user" in withWebDriver { implicit driver =>
+        val listPageAsUser1 = signIn(Config.Users.harry)
+        listPageAsUser1.signOut()
+        val listPageAsUser2 = signIn(Config.Users.ron)
+        assert(listPageAsUser2.getUser().startsWith(Config.Users.ron.email))
```

Comment:
I split them up to respect the principle of "one assert / test one _thing_". Do you think it is okay to combine them?